gcp/observability/logging_test.go

```diff
@@ -671,7 +671,7 @@ func (s) TestClientRPCEventsTruncateHeaderAndMetadata(t *testing.T) {
 			Authority:   ss.Address,
 			SequenceID:  1,
 			Payload: payload{
-				Metadata: map[string]string{"key1": "value1"},
+				Metadata: map[string]string{},
```

Comment:
This is definitely unfortunate that we don't get anything here.  It would be better if we could accept either header.  There should be a way to do that in the `cmp` package.

Reply:
Ok, I looked through cmp/cmpopts, and I couldn't find an explicit option that had the behavior you desire. The solution I came up with is to ignore the metadata in the cmp.Diff, and then compare with an or explicitly afterward.